content/hi/_index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+abstract: "स्वचालित कंटेनर डिप्लॉयमेंट, स्केलिंग और प्रबंधन"
+cid: home
+sitemap:
+  priority: 1.0
+---
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+[कुबेरनेट्स]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}), जो K8s के रूप में भी जाना जाता है, कंटेनरीकृत एप्लीकेशन के डिप्लॉयमेंट, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन-सोर्स सिस्टम है।
+
+यह आसान प्रबंधन और खोज के लिए लॉजिकल इकाइयों में एक एप्लीकेशन बनाने वाले कंटेनरों को समूहित करता है। कुबेरनेट्स [Google में उत्पादन कार्यभार चलाने के 15 वर्षों के अनुभव](http://queue.acm.org/detail.cfm?id=2898444), पर निर्माणित है, जो समुदाय के सर्वोत्तम-नस्लीय विचारों और प्रथाओं के साथ संयुक्त है।
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="scalable" %}}
+#### अद्भुत स्केल क्षमता <!--Planet Scale-->
+
+उन्हीं सिद्धांतों पर डिज़ाइन किया गया है जो Google को एक सप्ताह में अरबों कंटेनर चलाने की अनुमति देता है, कुबेरनेट्स आपकी ऑप्स टीम को बढ़ाए बिना स्केल कर सकता है।
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="blocks" %}}
+#### कभी नहीं बढ़ना <!--Never Outgrow-->
+
+चाहे स्थानीय स्तर पर परीक्षण हो या वैश्विक उद्यम चलाना, कुबेरनेट्स आपके एप्लीकेशन को लगातार और आसानी से वितरित करने के लिए आपके साथ बढ़ता है, चाहे आपकी आवश्यकता कितनी भी जटिल क्यों न हो।
```

Comment:
( आसानी से वितरित करने के लिए आपके साथ बढ़ता है ) here it means that "it grows with you" but in place of that we need to say that "it grows its flexibility" so can you reconstruct this statement